src/store/global/global.thunks.ts

```diff
@@ -964,7 +1027,73 @@ function fetchTokensPrice(): AppThunk {
     return adapters.priceUpdater
       .getTokensPrice()
       .then((res: Token[]) => dispatch(globalActions.loadTokensPriceSuccess(res)))
-      .catch(() => globalActions.loadTokensPriceFailure("An error occured loading token."));
+      .catch((error: unknown) => {
+        const errorMsg = adapters.parseError(
+          error,
+          "An error occurred on src/store/global/global.thunks.ts:fetchTokensPrice"
+        );
+        dispatch(globalActions.loadTokensPriceFailure(errorMsg));
+        dispatch(
+          openSnackbar({
+            message: {
+              type: "error",
+              raw: error,
+              parsed: errorMsg,
+            },
+          })
+        );
+      });
+  };
+}
+
+enum ReportErrorFormEntries {
```

Comment:
I would move this to a `const` inside the `constants.js` file, so we can find and replace the values easily

Reply:
Yeah makes sense. Done.